Add CSM Vulns limitations for Windows (#26641)
DataDog/documentation: content/en/security/cloud_security_management/vulnerabilities/hosts_containers_compatibility.md

| Ubuntu                   | All versions supported by Canonical                 | apt/dpkg                  | {{< X >}}         | {{< X >}}         |
| Windows                  | Windows Server 2016/2019/2022, Windows 10 and later | Windows OS                |                   | {{< X >}}         |

{{% collapse-content title="Windows limitations" level="h4" %}}

> - Datadog detects vulnerabilities in Windows by identifying the Windows version and the security KB updates that are installed. With this information, it can determine which vulnerabilities a Windows host is subject to and which updates have been released to address them. However, some KB updates are cumulative and contain other KB updates, which might cause Datadog to misidentify which updates have been installed.

Reviewer: 1. Could we make it slightly shorter. I'm afraid it could feel heavy
2. KB might not be obvious to everyone -> could we have 1 occurence that it mean Knowledge Base?

Author: @cyrbouchiat Sure! It now reads:
> - Datadog detects vulnerabilities in Windows by identifying the Windows version and installed security knowledge base (KB) updates to address vulnerabilities associated with that version. However, some KB updates are cumulative and contain other KB updates, which might cause Datadog to misidentify which updates have been installed.

Let me know if that works better :)

Reviewer 2: Looks like there are two files with KB in them and you only spelled it out in one instance... is that shortcode only getting used in this page before the mentions of KB, or does this page need that change too?

Author: Wow, a weekend really does make you forget what you were doing before. Thank you for flagging! Just applied the change to the other file.

> - Datadog detects vulnerabilities in Windows by identifying the Windows version, and the security KB updates that have been installed. With this information, it can determine which vulnerabilities a Windows host is subject to, and which updates have been released to address them. However, some KB updates can be cumulative and contain other KB updates, which may cause Datadog to misidentify which updates have been installed.

Reviewer 2: ```suggestion
- Datadog detects vulnerabilities in Windows by identifying the Windows version and the security KB updates that are installed. With this information, it can determine which vulnerabilities a Windows host is subject to and which updates have been released to address them. However, some KB updates are cumulative and contain other KB updates, which might cause Datadog to misidentify which updates have been installed.
```

| Windows                  | Windows Server 2016/2019/2022, Windows 10 and later | Windows OS                |                   | {{< X >}}         |

{{% collapse-content title="Windows limitations" level="h4" %}}

> - Datadog detects vulnerabilities in Windows by identifying the Windows version and installed security knowledge base (KB) updates to address vulnerabilities associated with that version. However, some KB updates are cumulative and contain other KB updates, which might cause Datadog to misidentify which updates have been installed.

Reviewer 2: ```suggestion
- Datadog detects vulnerabilities in Windows by identifying the Windows version and the installed security knowledge base (KB) updates to address vulnerabilities associated with that version. However, some KB updates are cumulative and contain other KB updates, which might cause Datadog to misidentify which updates have been installed.
```
